WhaleSpotting/Services/SightingsService.cs

```diff
@@ -64,5 +65,22 @@ public SightingResponseModel CreateSighting(SightingRequestModel sightingRequest
 
             return new SightingResponseModel(newSighting);
         }
+
+        public async Task<SightingResponseModel> ConfirmSighting(int id)
+        {
+            var sighting = await _context.Sightings
+                //TODO include user
```

Comment:
Do we need to include the user in the future?

Reply:
We wanted for every sighting to have the UserId for the user who posted that sighting.
And for our profile page to show the sightings of that user.
Currently we are just passing the  UserId when creating sightings. Do you think it is enough? Or should we connect them?

Reply:
We won't need to return the actual user for just a confirmation of sighting is my thought. For the full return we may want to, but here I would suggest not